<?xml version="1.0"?>
<launch>
    <include file="$(find assign_2)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="avg_window" value= "5" />
    <node pkg="assign_2" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assign_2" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assign_2" type="target_service.py" name="target" />
    <node pkg="assign_2" type="stats_service.py" name="stats"  />
    <node pkg="assign_2" type="bug_as.py" name="bug_action_server"  />
	<node pkg="assign_2" type="bug_ac.py" name="bug_action_client" output="screen" />
</launch>
